nexus/operation.go

```diff
@@ -15,6 +15,31 @@ import (
 //	)}
 type NoValue *struct{}
 
+// OperationOptions contains the general options for an operation, across different handlers.
+//
+// NOTE: Experimental
+type OperationOptions struct {
```

Comment:
Not loving the name `Options` but I can see that it's consistent across all methods and every method has its own `Options` struct.

Maybe we should call this `MiddlewareOptions` or `MiddlewareInfo`?

Reply:
Alternatively we could call this something like `CommonOperationOptions` and embedded it in every other operation struct

Reply:
No need to embed it, the options are used both in the client and the server and this information would be redundant.

Reply:
Another option is `HandlerInfo`, does that seem better to you?

Reply:
hmm it is OK